traffic_ops/testing/api/v14/cdnfederations_test.go

```diff
@@ -82,7 +82,7 @@ func UpdateTestCDNFederations(t *testing.T) {
 		log.Debugf("GET Response: %s\n", bytes)
 
 		if resp2.Response[0].CName == nil {
-			log.Errorf("CName is nil after updating\n")
+			log.Errorln("CName is nil after updating")
```

Comment:
should this just be `log.Error` as you did in other spots?

Reply:
There is no `log.Error` function in `lib/go-log`. See https://godoc.org/github.com/apache/trafficcontrol/lib/go-log.

I believe nearly all the other similar changes this PR makes are calling `t.Error` where `t` is a `*testing.T`.

Reply:
ahh shoot I didnt even see this was a log instead of t.Error my bad :)

Reply:
No worries, I did the same thing while working on this PR.